web/pages/groups/index.tsx

```diff
@@ -44,7 +44,10 @@ const Groups = () => {
         // Do not search with empty query
         if (!searchTerm) return;
 
-        const queryParams: GroupSearchGetReq = { searchTerm };
+        const queryParams: GroupSearchGetReq = {
+            searchTerm,
```

Comment:
I don't think you meant to change this back to using `n`? If you did, lmk, otherwise could you try using `{ searchTerm }` by itself (like before this change) and see if it works?